app/uk/gov/hmrc/economiccrimelevyregistration/viewmodels/AddressViewModel.scala

```diff
@@ -45,13 +45,15 @@ object AddressViewModel {
 
   def html(eclAddress: GetCorrespondenceAddressDetails): String = eclAddressToSeq(eclAddress)
     .filter(_.isDefined)
-    .map(value => HtmlFormat.escape(value.get))
-    .mkString("<br/>")
+    .map(value => HtmlFormat.raw(value.get.prependedAll("<p id=\"address\">")))
+    .mkString("</p>")
+    .appendedAll("</p>")
```

Comment:
is the appendAll needed in addition to the mkString?

Reply:
Yes,  mkString adds </p> to each element except of the last one, This is snipper from documentation:

"_Displays all elements of this collection in a string using a separator string
Example:
List(1, 2, 3).mkString("|") = "1|2|3"_
"
As you can see there | is not added on last element. In our case missing </p> on last element would cause rendering issues for PDF.